pkg/kubelet/container/helpers.go

```diff
@@ -134,6 +135,14 @@ func HashContainerWithoutResources(container *v1.Container) uint64 {
 	return uint64(hashWithoutResources.Sum32())
 }
 
+// HashAuth - returns a hash code for a CRI pull image auth
+func HashAuth(auth *runtimeapi.AuthConfig) string {
+	hash := fnv.New64a()
+	authJSON, _ := json.Marshal(auth)
```

Comment:
can you add a comment as to why we ignore the error here?

Reply:
Updated. log an error and return "" here.